src/contexts/Product/ProductContext.tsx

```diff
@@ -25,14 +25,7 @@ export const ProductContextProvider = ({
     const [products, setProducts] = useState<Product[]>([]);
 
     useEffect(() => {
-        import('@data/products')
-            .then((data) => {
-                setProducts(data.products);
-            })
-            .catch((error) => {
-                // eslint-disable-next-line no-console
-                console.error('Error loading products:', error);
-            });
+        setProducts(PRODUCTS);
     }, []);
 
     return (
```

Comment:
although we dont require it, but add a means to update the array of products as well.